scripts/update/0.5-0.6/cluster.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# This script:
+# 1) Appends the label `lokomotive.alpha.kinvolk.io/bgp-enabled=true` in the env file for the nodes
+#    running MetalLB.
+# 2) Update the image tag of Kubelet.
+# 3) Update etcd if it is a controller node.
+
+set -euo pipefail
+
+mode=$1
+
+function run_on_host() {
+  nsenter -a -t 1 /bin/sh -c "${1}"
+}
+
+updated_kubelet_config=false
+
+function is_packet_environment() {
+  if grep -i packet /run/metadata/flatcar > /dev/null; then
+    packet_environment=true
+  else
+    packet_environment=false
```

Comment:
Since this is a global variable, maybe define it at false on top and then override it to true here?